fidouafclient/app/src/main/java/org/ebayopensource/fidouafclient/op/Reg.java

```diff
@@ -38,17 +38,22 @@
 public class Reg {
 	
 	private Gson gson = new GsonBuilder().disableHtmlEscaping().create();
-	
+
 	public String getUafMsgRegRequest (String username, String appId){
```

Comment:
Thank you,  Emerson.

This was a very good review of what is happening. I believe if app id overwrite is removed from the Reg class, then we will be fine.

If you want you could try to change this, and include in the PR.

Thanks,
Neb.